src/Servers/HttpSys/src/IHttpSysRequestInfoFeature.cs

```diff
@@ -7,8 +7,8 @@ namespace Microsoft.AspNetCore.Server.HttpSys;
 
 /// <summary>
 /// This exposes the Http.Sys HTTP_REQUEST_INFO extensibility point as opaque data for the caller to interperate.
-/// https://docs.microsoft.com/en-us/windows/win32/api/http/ns-http-http_request_v2
-/// https://docs.microsoft.com/en-us/windows/win32/api/http/ns-http-http_request_info
+/// <see href="https://docs.microsoft.com/en-us/windows/win32/api/http/ns-http-http_request_v2"/>
+/// <see href="https://docs.microsoft.com/en-us/windows/win32/api/http/ns-http-http_request_info"/>
```

Comment:
These will render back to back in Intellisense popups (existing issue), recommend a command or something to separate